app/streaming/trace/trace.go

```diff
@@ -0,0 +1,33 @@
+package trace
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	"github.com/provenance-io/provenance/app/streaming/trace/service"
+	"github.com/provenance-io/provenance/internal/streaming"
+	"github.com/spf13/cast"
+)
+
+const (
+	// TomlKey is the top-level TOML key for TraceStreamingService configuration
+	TomlKey = "trace"
+
+	// PrintDataToStdoutParam is the TraceStreamingService flag that logs full streamed data at debug level
+	PrintDataToStdoutParam = "print_data_to_stdout"
```

Comment:
In Tendermint `v0.35`, they changed all config field names to use dashes instead of underscores. Cosmos-SDK already uses only dashes in their config entries. Unless there's a good reason to use underscores here, we probably want to go with dashes instead:
```suggestion
	PrintDataToStdoutParam = "print-data-to-stdout"
```
The documentation would also need updating.